### PR TITLE
bitwarden: add Openstack cluster secrets

### PR DESCRIPTION
**Pull request by bbguimaraes**

### CODE REVIEW

File: ci-operator/SECRETS.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ secrets currently exist:
...
 | `system-docker-config.json`   | Root/node/system level docker config.json file, currently holding access registry.redhat.io |
 
+#### `cluster-secrets-openstack`
+
+|        Key        | Description |
+| ----------------- | ----------- |
+| `credentials`     | TODO |
+| `clouds.yaml`     | TODO |
+| `ssh-publickey`   | TODO |
+| `pull-secret`     | Credentials for pulling OpenShift images from Quay. |
```
Comment (bbguimaraes) [inline ci-operator/SECRETS.md]: @flaper87: please fill in this section (either in a comment here and I'll include it or in a separate PR).

Comment (flaper87) [reply ci-operator/SECRETS.md]: `clouds.yaml` -> credentials for the openstack cloud. See `https://docs.openstack.org/python-openstackclient/pike/configuration/index.html`

`ssh-privatekey` - > Private half of the SSH key, for connecting to OpenStack Nova VMs.
`ssh-publickey` -> Public half of the SSH key, for connecting to OpenStack Nova VMs.


TBH, I've no idea what `credentials` is for. It's unclear to me if we're still using it but I believe it's got the same value as the `clouds.yaml` key. I think we can skip it for now

File: ci-operator/populate-secrets-from-bitwarden.sh [hunk 2]
```diff
@@ -212,6 +212,14 @@ update_secret generic "cluster-secrets-${target_cloud}"                       \
 	"$( format_field_value telemeter "Telemeter Token" "telemeter-token" )"
 oc label secret "cluster-secrets-${target_cloud}" "ci.openshift.io/managed=true"
 
+target_cloud="openstack"
+update_secret generic "cluster-secrets-${target_cloud}"                       \
+	--from-literal=pull-secret="$(merge_pull_secrets)"                          \
+	"$( format_attachment "openstack" clouds.yaml )"                            \
+	"$( format_attachment "openstack" clouds.yaml credentials )"                \
+	"$( format_attachment "openstack" ssh-publickey )"
```
Comment (flaper87) [inline ci-operator/populate-secrets-from-bitwarden.sh]: I think we're missing the private key

Comment (bbguimaraes) [reply ci-operator/populate-secrets-from-bitwarden.sh]: I also found that strange, but it is also missing from the secret currently in the cluster:

```yaml
apiVersion: v2
data:
  clouds.yaml: XXX
  credentials: XXX (same as clouds.yaml)
  pull-secret: XXX
  ssh-publickey: XXX
kind: Secret
metadata:
  name: cluster-secrets-openstack
  namespace: ci
type: Opaque
```

Comment (flaper87) [reply ci-operator/populate-secrets-from-bitwarden.sh]: Can we use the same values as for AWS? That way we can make ssh'ing into this CI environments more consistent. I believe you guys already have a shared private key that you use for this envs

**Comment by bbguimaraes** [comment]
Documentation pdated, `credentials` removed, public/private keys added, configuration applied to the cluster for testing.

**Comment by flaper87** [comment]
I've tested the credentials and it seems to work fine. I can see vms being created in our CI environment. The job still needs some work but I'm on it.

**Pull request merged**